src/main/java/com/eternalcode/discordapp/DiscordApp.java

```diff
@@ -61,6 +67,12 @@ public static void main(String... args) throws InterruptedException {
         configManager.load(databaseConfig);
         configManager.load(experienceConfig);
 
+        DataManager dataManager = new DataManager(new File("data"));
+        UsersVoiceActivityData usersVoiceActivityData = new UsersVoiceActivityData();
+        dataManager.load(usersVoiceActivityData);
+
+        usersVoiceActivityData.usersOnVoiceChannel.put(0L, Instant.now().toEpochMilli());
```

Comment:
Po to Bozia Instant dała żeby go używać, a nie Longi. Tylko trzeba dodać composer do cdn od instanta. 
Pomocne:
Instant#toString() : String 
Instant.parse(String) : Instant

Reply:
Long zostanie ze względu na operacje matematyczne - obliczenie długości czasu spędzonego na kanale. Przetramsformowanie na stringa dodałoby trochę bezsensownego kodu.

Reply:
Proszę używać API Instanta i Duration, po to istnieje to żeby nie robić operacji matematycznych tylko wykonywać sobie metody do liczenia punktu w czasie i okresu czasu.